onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -3307,10 +3329,13 @@ Status TensorrtExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphView
       // Set precision
       if (trt_state->fp16_enable && trt_state->int8_enable) {
         trt_config->setFlags(1U << static_cast<uint32_t>(nvinfer1::BuilderFlag::kFP16) | 1U << static_cast<uint32_t>(nvinfer1::BuilderFlag::kINT8));
-      } else if (trt_state->fp16_enable) {
+      trt_state->strong_typing_enable = false;} else if (trt_state->fp16_enable) {
```

Comment:
The `trt_state->strong_typing_enable` won't used in compute_info.compute_func. Do we need this here?